server/admin_management/api/app/routers/router_access_control_privilege.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from fastapi import APIRouter, Depends, Request
+from sqlalchemy.orm import Session
+from typing import List
+
+from api.app.models import model as models
+from api.app.routers.router_guards import (
+    get_current_requester,
+    authorize_by_application_role,
+    enforce_self_grant_guard,
+)
+from api.app import database, jwt_validation, schemas
+from api.app.schemas import Requester
+from api.app.services.access_control_privilege_service import (
+    AccessControlPrivilegeService,
+)
+from api.app.services.user_service import UserService
+from api.app.services.role_service import RoleService
+from api.app.utils.audit_util import AuditEventLog, AuditEventOutcome, AuditEventType
+
+LOGGER = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.post(
+    "",
+    response_model=List[schemas.FamAccessControlPrivilegeCreateResponse],
+    dependencies=[
+        Depends(
+            authorize_by_application_role
+        ),  # only app admin can do this, get application by role
+        Depends(enforce_self_grant_guard),
+    ],
+)
+def create_access_control_privilege(
+    access_control_privilege_request: schemas.FamAccessControlPrivilegeCreateRequest,
+    request: Request,
+    db: Session = Depends(database.get_db),
+    token_claims: dict = Depends(jwt_validation.authorize),
+    requester: Requester = Depends(get_current_requester),
+):
```

Comment:
Hi Catherine, now I understand, the new endpoint you are adding is different than normal ones.
You are essentially adding a "**bulk**" creation of REST resource**s** (**ACCESS_CONTROL_PRIVLIEGE**) for the endpoint, not as previously one at a time.
Can we use method name as "create_access_control_privilege_**many/bulk**" and add some method comment for it (as initially I wasn't aware by this method name and got confused and realized later you are creating bulk endpoint). Thanks Catherine!
Or, @basilv , maybe you have better naming or comment on bulk design on POST endpoint?